### PR TITLE
Update documentation towards the 2.2 release

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Adds README and CHANGELOG to the repository. I need to release #11 and #9, so I would like to have these common files in place.

Docs are mostly taken from https://wiki.jenkins.io/display/JENKINS/Instance+Identity which can be replaced by redirect after the merge

@jenkinsci/java10-support + @kohsuke @stephenc who contributed changes

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
I will go forward with the release for now, happy to edit descriptions if there is any feedback

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Each Jenkins instance maintains an RSA private/public key pair
+that can be used to uniquely identify Jenkins.
+This information is called "instance identity".
+
+From outside, the public key can be obtained by sending the GET request
+to the top page of Jenkins,
+and look for the _X-Instance-Identity_ header in the response.
+This header is always available, even if the response is 401 access denied
+(which can happen if Jenkins is protected via security.)
```
Comment (daniel-beck) [inline README.md]: The problem is the redirect to `/login` that doesn't have those headers (IIRC). Ask @rtyler some time how much he enjoyed figuring this out for the CLI docs: https://jenkins.io/doc/book/managing/cli/#ssh

Comment (oleg-nenashev) [reply README.md]: It was a copy-paste from Wiki. Pull requests are welcome

Comment (daniel-beck) [reply README.md]: Oops, missed that.

**Pull request merged**